Code/Framework/AzCore/AzCore/Math/IntersectSegment.cpp

```diff
@@ -406,6 +406,13 @@ namespace AZ
         return ISECT_RAY_AABB_ISECT;
     }
 
+    Intersect::RayAABBIsectTypes Intersect::IntersectRayAABB2(const Vector3& rayStart, const Vector3& dirRCP, const Aabb& aabb)  
+    {
+        [[maybe_unused]] float start, end;
```

Comment:
you don't have to change this and apologies I wasn't clear but I intended

```
float start_unused, end_unused;
```

Just to show the variables don't have any real purpose